### PR TITLE
Log a warning if users are trying to play a touchscreen game in docked mode

### PR DESCRIPTION
**Pull request by lostromb**
This is mostly to address cases of "user error" and to avoid bug reports like this #3922.

If the console is set to docked mode and the user is trying to interact with the touchscreen, show a warning on the log that this is physically impossible on real hardware and may not work.

### CODE REVIEW

File: Ryujinx.Input/HLE/TouchScreenManager.cs [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 using Ryujinx.HLE.HOS.Services.Hid;
 using Ryujinx.HLE.HOS.Services.Hid.Types.SharedMemory.TouchScreen;
+using Ryujinx.HLE.HOS.SystemState;
 using System;
 
 namespace Ryujinx.Input.HLE
 {
     public class TouchScreenManager : IDisposable
     {
+        private static readonly TimeSpan _invalidTouchWarningInterval = TimeSpan.FromSeconds(5);
```
Comment (lostromb) [inline Ryujinx.Input/HLE/TouchScreenManager.cs:12]: maybe need to reformat the name of this var since it's a const

Comment (lostromb) [reply Ryujinx.Input/HLE/TouchScreenManager.cs:12]: actually nah I guess the style is all right, based on consts in other files...

**Comment by german77** [comment]
You might want to cleanup the commits to just one

**Comment by lostromb** [comment]
> You might want to cleanup the commits to just one

yeah my master branch is garbage. I can rebase I guess...

**Comment by lostromb** [comment]
barf!

**Pull request closed without merging**